pkg/kubelet/cri/remote/remote_image.go

```diff
@@ -84,14 +84,14 @@ func (r *remoteImageService) ImageStatus(image *runtimeapi.ImageSpec) (*runtimea
 		Image: image,
 	})
 	if err != nil {
-		klog.Errorf("ImageStatus %q from image service failed: %v", image.Image, err)
+		klog.ErrorS(err, "Get image status from image service failed", "image", image.Image)
 		return nil, err
 	}
 
 	if resp.Image != nil {
 		if resp.Image.Id == "" || resp.Image.Size_ == 0 {
 			errorMessage := fmt.Sprintf("Id or size of image %q is not set", image.Image)
-			klog.Errorf("ImageStatus failed: %s", errorMessage)
+			klog.ErrorS(nil, "ImageStatus failed", "err", errorMessage)
```

Comment:
Can we return/wrap the errorMessage rather than passing nil and adding this as an annotation?